qiskit_ibm_runtime/ibm_backend.py

```diff
@@ -239,16 +239,6 @@ def __getattr__(self, name: str) -> Any:
                 "'{}' object has no attribute '{}'".format(self.__class__.__name__, name)
             )
 
-    def _get_properties(self, datetime: Optional[python_datetime] = None) -> None:
-        """Gets backend properties and decodes it"""
-        if datetime:
-            datetime = local_to_utc(datetime)
-        if datetime or not self._properties:
-            api_properties = self._api_client.backend_properties(self.name, datetime=datetime)
-            if api_properties:
-                backend_properties = properties_from_server_data(api_properties)
-                self._properties = backend_properties
-
     def _get_defaults(self, refresh: bool = False) -> None:
```

Comment:
This can be a separate PR, but I noticed there is a `_get_defaults()` which seems to do the same thing as `defaults()`

Reply:
I'll handle this in a separate PR

Reply:
We can use #1910